Formula/med-file.rb

```diff
@@ -0,0 +1,48 @@
+class MedFile < Formula
+  desc "Modeling and Data Exchange standardized format"
+  homepage "http://www.salome-platform.org"
+  url "http://files.salome-platform.org/Salome/other/med-3.2.0.tar.gz"
+  sha256 "d52e9a1bdd10f31aa154c34a5799b48d4266dc6b4a5ee05a9ceda525f2c6c138"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "swig" => :build
+  depends_on "hdf5"
+  depends_on "python"
```

Comment:
system python should be sufficient
```
depends_on "python" if MacOS.version <= :snow_leopard
```

Reply:
matplotlib is a site-package library.  I thought we weren't supposed to mix python interpreters with brewed site-packages?

Reply:
see tvnamer for an example

Reply:
sorry, was thinking `matplotlib` (another I am working on).  `med-file` is a file format library that includes bindings for Python and Fortran.  med-file is not available via pip.

That said, is `tvnamer` the exemplar for brewing a pip installable site-package?  Does that mean that I should completely rewrite the legacy [matplotlib](https://github.com/Homebrew/homebrew-science/blob/19f75951641d3a5e70ea105f76a6a77bc0553d07/matplotlib.rb) to take the same approach?

Reply:
unlikely to accept matplotlib since it's a library

Reply:
:(

Reply:
it's available via pip, can be vendored in other formulae as needed, and takes like a minute to build, so I don't see what value a formula adds. when we killed pillow it had high analytics numbers and yet there was barely a peep

Reply:
Ok.  Glad this came up so I didn't waste a lot of time on it. 

What do you mean by "vendored in other formulae"?  Can you point me to an example?